src/Sylius/Bundle/AdminBundle/templates/product/show/content/page_body/details/configurable_product/details/pricing.html.twig

```diff
@@ -0,0 +1,32 @@
+{% set variant = hookable_metadata.context.variant %}
+{% set channels = hookable_metadata.context.channels %}
+
+<div class="card mb-3">
+    <div class="card-header">
+        <div class="card-title">{{ 'sylius.ui.pricing'|trans }}</div>
+    </div>
+    <div class="table-responsive">
+        <table class="table card-table table-vcenter">
+            <thead>
+            <tr>
+                <th>{{ 'sylius.ui.channel'|trans }}</th>
+                <th class="text-end">{{ 'sylius.ui.price'|trans }}</th>
+                <th class="text-end">{{ 'sylius.ui.original_price'|trans }}</th>
+                <th class="text-end">{{ 'sylius.ui.lowest_price_before_discount'|trans }}</th>
+                <th class="text-end">{{ 'sylius.ui.discounted_by'|trans }}</th>
+                <th></th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for channel_pricing in variant.channelPricings %}
+                {% set channel = attribute(channels, channel_pricing.channelCode) %}
+                {% set currency_code = channel.baseCurrency.code %}
+
+                <tr {{ sylius_test_html_attribute('variant-pricing', '%s.%s'|format(channel.code, variant.code)) }} >
+                    {% hook 'pricing' with { variant, channel, channel_pricing, currency_code } %}
+                </tr>
+            {% endfor %}
```

Comment:
This is a bit problematic, I can add more data via the hook, but I cannot label them, since headers are static.